### ISSUE TITLE
Keyboard shortcut for editing / adding new tags

### ISSUE THREAD

**Issue by joshlvmh** [johannesjo/super-productivity#808]
Would it be possible to implement a keyboard shortcut for interaction with tags?

**Comment by johannesjo** [comment]
Hey there! Thanks for the suggestion. Should be a good issue for people being new to the project! Meanwhile did you know that you can edit tasks by pressing enter while they are focused? This way you can add tag names like so: `Task title #tag #other`

**Comment by joshlvmh** [comment]
Thanks for this! Good to know. Look forward to future updates.

**Comment by rmpsc** [comment]
@johannesjo Hi! New to PRs here. May I work on this issue?

**Comment by johannesjo** [comment]
@rmpsc yes, please! :)

Let me know if you have any questions!

**Comment by rmpsc** [comment]
@johannesjo To make sure, would this shortcut intend on taking the user to the first tag on the list? (if any)

**Comment by johannesjo** [comment]
Sorry for taking my time with the reply. I was pretty busy this week. 

Shortcut related stuff can be found here:
```
src/app/core-ui/shortcut/shortcut.service.ts
src/app/features/config/default-global-config.const.ts
src/app/features/config/form-cfgs/keyboard-form.const.ts
```

I would think that a shortcut would open the Dialog. What do you think?

**Comment by rmpsc** [comment]
I agree on opening the Dialog for whichever task is in focus. Thank you for the shortcut locations. I'll look thru them and get started as soon as I can!

**Comment by johannesjo** [comment]
@rmpsc Great! Let me know if you have any other questions!

**Comment by rmpsc** [comment]
@johannesjo I'm about finished although I'm having trouble finding the file that shows the default keyboard character/s for the shortcut. I currently have the shortcut set to 'z' in default-global-config.const.ts, but this is not what shows in the settings menu. Where would the correct file be located? Thank you
![image](https://user-images.githubusercontent.com/49537451/111090527-41bdb780-84ed-11eb-85fe-8f8d98b336b4.png)

**Comment by johannesjo** [comment]
The default config is located here as you mention:
```
src/app/features/config/default-global-config.const.ts
```
It just might be the case, that the app already saved an empty string for the key as value. To test how the config screen would look like to new users, you can open up an inkognito tab in your browser and check there.

**Comment by rmpsc** [comment]
Just checked and that was the case! Thanks for the tip. I'll look for translations for "Edit Tags" in other languages and insert those into the appropriate JSON files. Are there any other preferences you would like before I pr? Possibly a specific location of Edit Tags on the config page or a preferred default keybind?

**Comment by johannesjo** [comment]
Hi @rmpsc no need to make those translations manually (apart from English and the languages you like to use yourself ;)). I autogenerate all translations once in a while.

"Edit Tags" should ideally be shown after the other task shortcuts I think.

**Comment by rmpsc** [comment]
Just submitted the pr. Let me know what you think :)
